docs/admin/auth.md

```diff
@@ -326,11 +432,35 @@ Users who are not in a matching group will see the following error:
 
 ![Unauthorized group error](../images/admin/group-allowlist.png)
 
-## Role sync (enterprise)
+## Role sync (enterprise) (premium)
 
 If your OpenID Connect provider supports roles claims, you can configure Coder
-to synchronize roles in your auth provider to deployment-wide roles within
-Coder.
+to synchronize roles in your auth provider to roles within Coder.
+
+There are two ways you can configure group sync:
```

Comment:
There are 2 ways to do role sync. Server Flags assign **site wide** roles, and runtime org role sync assigns **organization roles**